domains/abhay.json

```diff
@@ -0,0 +1,11 @@
+{
+        "owner": {
+           "username": "uniquepersun",
+           "email": "abhay.i0am@gmail.com",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.WIRf3W3gkAD5c_yyczIieWIABVFKWjmCVV0l_jrxtMMOnT2mWlAELQAVfwDsjz2XEX6NcrWmCEaTEYC0aVCeWEgSjnl_r077RRwQuAOED2ICK-M5WvKDPdUuPlChYVtpbRiByY1nmFWUp5DGn7Z2faQrla_7Ji2e924mSSJppOeBuhcDZ0mZ0Mc3Z8X4DWgQ5xBbKFoREVkyuPSs9Y6Q0-lZsFZijj9khQsQDbDgZT8v3Fv9swLkqAdt-ZMBr7ZdvJSkBbtWj8L038WAZ6dSg-ix3xpkluU97jQt6kLkOyRYY5MKDFoSkxraBumZA9-YDB3pDL_2yAZ23cvHywnPDg.kU_H99RhrpznpQWwrfJ_Aw.yuZV5PvTwNvz5T1JlXOnGw7JjHcUDj_KdD9XGpvQ2rQQ9twNQJESMqj55Jxbgx2jX6mzJsRWmlJDPkE3eSkMQOv3AJuxuWdX4kGpREkxeixFGB8N1oviCe6z0JoYfRuW.GdyFBUkFmKP1fVuGM1CdCA"
+        },
+    
+        "record": {
+            "CNAME": ""
```

Comment:
You must have a complete website, and see https://www.is-a.dev/docs/domain-structure/